lua/cmp/utils/async.lua

```diff
@@ -276,7 +276,7 @@ end
 
 -- This will yield when called from a coroutine
 function async.yield(...)
-  if not coroutine.isyieldable() then
+  if coroutine.isyieldable ~= nil and not coroutine.isyieldable() then
```

Comment:
This can be safely replaced by:

```lua
  if coroutine.running() == nil then
```

That way we don't need `coroutine.isyieldable`